plugins/legacy-preset-chart-big-number/src/BigNumber/controlPanel.tsx

```diff
@@ -56,6 +56,18 @@ const config: ControlPanelConfig = {
           },
         ],
         ['y_axis_format'],
+        [
+          {
+            name: 'show_timestamp',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show Timestamp'),
+              renderTrigger: true,
+              default: false,
+              description: t('Whether to display the timestamp'),
+            },
+          },
+        ],
```

Comment:
Could we also add a date formatter control that is made visible if `show_timestamp` is `true`? Here's a time formatter control from the ECharts timeseries charts: 
![image](https://user-images.githubusercontent.com/33317356/128854044-3cf38d72-59a5-44e4-9425-a1e681809a8f.png)

Reply:
added!